src/Xamarin.Android.Build.Tasks/Microsoft.NET.Workload.Android/WorkloadManifest.in.json

```diff
@@ -18,6 +18,7 @@
       "alias-to": {
         "osx-x64": "Microsoft.Android.Sdk.osx-x64",
         "osx-arm64": "Microsoft.Android.Sdk.osx-x64",
+        "win-x86": "Microsoft.Android.Sdk.win-x64",
         "win-x64": "Microsoft.Android.Sdk.win-x64",
         "linux-x64": "Microsoft.Android.Sdk.linux-x64"
```

Comment:
Should we instead rename these to:
* Microsoft.Android.Sdk.Windows
* Microsoft.Android.Sdk.MacOS
* Microsoft.Android.Sdk.Linux

They aren't going to map to RIDs, because we probably want the Mac & Windows packages to work on multiple architectures.

Reply:
I think renaming these as suggested makes sense 👍

Reply:
I used the name `Microsoft.Android.Sdk.Darwin`, so it matches `$(HostOS)`.

We'll see what breaks.